drivers/hmis/lib/form_data/records/service.json

```diff
@@ -0,0 +1,139 @@
+    },
+    {
+      "type": "STRING",
+      "required": false,
+      "link_id": "movingOnOtherType",
+      "text": "Moving On Other Type",
+      "field_name": "movingOnOtherType",
+      "enable_behavior": "ALL",
+      "enable_when": [
+        {
+          "question": "typeProvided",
+          "operator": "EQUAL",
+          "answer_code": "C2_MOVING_ON_ASSISTANCE_PROVIDED__OTHER_PLEASE_SPECIFY"
+        }
+      ]
+    },
+    {
+      "type": "CHOICE",
+      "link_id": "subTypeProvided3",
+      "required": false,
+      "text": "Sub Type Provided",
+      "pick_list_reference": "SUB_TYPE_PROVIDED_3",
+      "field_name": "subTypeProvided",
+      "enable_behavior": "ALL",
+      "enable_when": [
+        {
+          "question": "typeProvided",
+          "operator": "EQUAL",
+          "answer_code": "SSVF_SERVICE__ASSISTANCE_OBTAINING_VA_BENEFITS"
+        }
+      ]
+    },
+    {
+      "type": "CHOICE",
+      "link_id": "subTypeProvided4",
+      "required": false,
+      "text": "Sub Type Provided",
+      "pick_list_reference": "SUB_TYPE_PROVIDED_4",
+      "field_name": "subTypeProvided",
+      "enable_behavior": "ALL",
+      "enable_when": [
+        {
+          "question": "typeProvided",
+          "operator": "EQUAL",
+          "answer_code": "SSVF_SERVICE__ASSISTANCE_OBTAINING_COORDINATING_OTHER_PUBLIC_BENEFITS"
+        }
+      ]
+    },
+    {
+      "type": "CHOICE",
+      "link_id": "subTypeProvided5",
+      "required": false,
+      "text": "Sub Type Provided",
+      "pick_list_reference": "SUB_TYPE_PROVIDED_5",
+      "field_name": "subTypeProvided",
+      "enable_behavior": "ALL",
+      "enable_when": [
+        {
+          "question": "typeProvided",
+          "operator": "EQUAL",
+          "answer_code": "SSVF_SERVICE__DIRECT_PROVISION_OF_OTHER_PUBLIC_BENEFITS"
+        }
+      ]
+    },
+    {
+      "type": "CURRENCY",
+      "required": false,
+      "link_id": "FAAmount",
+      "text": "FA Amount",
+      "field_name": "FAAmount",
+      "enable_behavior": "ANY",
+      "enable_when": [
+        {
+          "question": "typeProvided",
+          "operator": "EQUAL",
+          "answer_group_code": "HOPWA_FINANCIAL_ASSISTANCE"
+        },
+        {
+          "question": "typeProvided",
+          "operator": "EQUAL",
+          "answer_group_code": "SSVF_FINANCIAL_ASSISTANCE"
+        }
+      ]
+    },
+    {
+      "type": "CHOICE",
+      "link_id": "referralOutcome",
+      "required": false,
+      "text": "Referral Outcome",
+      "pick_list_reference": "REFERRAL_OUTCOME",
+      "field_name": "referralOutcome",
+      "enable_behavior": "ANY",
+      "enable_when": [
+        {
+          "question": "typeProvided",
+          "operator": "EQUAL",
+          "answer_group_code": "PATH_REFERRAL"
+        }
+      ]
+    },
+    {
+      "type": "DATE",
+      "link_id": "dateProvided",
+      "required": true,
+      "text": "Date Provided",
+      "field_name": "dateProvided"
+    }
```

Comment:
I wonder if this should go first?

Reply:
@eanders Happy to put it first if you would like

Reply:
I think we should leave it as it is.  After seeing the UI, I think it's probably correct.